src/Model/tasks.js

```diff
@@ -0,0 +1,8 @@
+const Data = [
+  // { completed: false, description: 'Refactor code to ES6 modules', id: 1 },
+  // { completed: false, description: 'Create default componets for the project', id: 2 },
+  // { completed: false, description: 'Extract code to specific components', id: 3 },
+  // { completed: false, description: 'Request a review', id: 4 },
+];
```

Comment:
- [ ] [OPTIONAL] It would be better if you remove these codes because they are no longer needed. This will help you make your code clean 😉

Reply:
Thank you @BertrandConxy, I'd effect the observations immediately.